src/k8-config/src/context.rs

```diff
@@ -25,71 +35,234 @@ fn load_cert_auth() -> String {
         .to_string()
 }
 
-pub struct Option {
```

Comment:
this break semver requirements.  we should keep existing API.  it can be mapped new one but old code should work.